client/my-sites/checkout/checkout-thank-you/redesign-v2/pages/plan-only.tsx

```diff
@@ -0,0 +1,93 @@
+import { isP2Plus } from '@automattic/calypso-products';
+import { translate } from 'i18n-calypso';
+import moment from 'moment';
+import ThankYouV2 from 'calypso/components/thank-you-v2';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { preventWidows } from 'calypso/lib/formatting';
+import { useSelector } from 'calypso/state';
+import { getSiteOptions } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import ThankYouPlanProduct from '../products/plan-product';
+import type { ReceiptPurchase } from 'calypso/state/receipts/types';
+
+interface PlanOnlyThankYouProps {
+	primaryPurchase: ReceiptPurchase;
+}
+
+export default function PlanOnlyThankYou( { primaryPurchase }: PlanOnlyThankYouProps ) {
+	const isMonthsOld = ( rawDate: string, months: number ) => {
+		if ( ! rawDate || ! months ) {
+			return false;
+		}
+
+		const parsedDate = moment( rawDate );
+		return moment().diff( parsedDate, 'months' ) > months;
+	};
+
+	const siteId = useSelector( getSelectedSiteId );
+	const siteSlug = useSelector( getSelectedSiteSlug );
+	const siteCreatedTimeStamp = useSelector(
+		( state ) => getSiteOptions( state, siteId ?? 0 )?.created_at
+	);
+
+	const footerDetails = [];
+
+	if ( isP2Plus( primaryPurchase ) ) {
+		footerDetails.push( {
+			key: 'footer-add-members',
+			title: translate( 'Go further, together' ),
+			description: translate(
+				'Invite people to the P2 to create a fully interactive work environment and start getting better results.'
+			),
+			buttonText: translate( 'Add members' ),
+			buttonHref: `/people/new/${ siteSlug }`,
+			buttonOnClick: () => {
+				recordTracksEvent( 'calypso_plan_thank_you_add_members_click' );
+			},
+		} );
+	} else if ( isMonthsOld( siteCreatedTimeStamp ?? '', 6 ) ) {
```

Comment:
```suggestion
	} else if ( isMonthsOld( siteCreatedTimeStamp, 6 ) ) {
```

in the isMonthsOld we have condition `if ( ! rawDate || ! months ) {`, so let's go the next way:
1) `const isMonthsOld = ( rawDate?: string, months?: number ) => {` // allow undefined
2) `} else if ( isMonthsOld( siteCreatedTimeStamp, 6 ) ) {` provide undefined instead of forcing empty string

Reply:
Good point! I also don't see why we wouldn't supply `months`, so I've updated the function as follows:

```
const isMonthsOld = ( rawDate?: string, months: number ) => {
	if ( ! rawDate ) {
		return false;
	}

	const parsedDate = moment( rawDate );
	return moment().diff( parsedDate, 'months' ) > months;
}
```

The optional parameter `rawDate` is really just meant to account for the case where `siteCreatedTimeStamp` is unexpectedly missing.